storio-sqlite-annotations/src/main/java/com/pushtorefresh/storio/sqlite/annotations/StorIOSQLiteColumn.java

```diff
@@ -35,4 +35,14 @@
      * @return true if column with {@code null} value should be ignored, false otherwise
      */
     boolean ignoreNull() default false;
+
+    /**
+     * Optional: Indicates version of database on which column was added. It is used for generating sql to create table.
+     * If version is 0 (default value) then code for creating column will be generated in createTable method.
+     * Other values will create column code in updateTable section only for databases that version is lower than this field
+     * Negative values are prohibited.
+     *
+     * @return version of database on which column was added.
+     */
+    int version() default 0;
```

Comment:
Should we start from `1`? Because it is the base value by [docs](https://developer.android.com/reference/android/database/sqlite/SQLiteOpenHelper.html#SQLiteOpenHelper(android.content.Context,%20java.lang.String,%20android.database.sqlite.SQLiteDatabase.CursorFactory,%20int))